packages/nouns-webapp/src/wagmi.ts

```diff
@@ -0,0 +1,43 @@
+import { find, pipe } from 'remeda';
+import { createConfig, http } from 'wagmi';
+import { mainnet, sepolia } from 'wagmi/chains';
+import { coinbaseWallet, injected, walletConnect } from 'wagmi/connectors';
+
+import { CHAIN_ID, WALLET_CONNECT_V2_PROJECT_ID } from './config';
+
+const activeChainId = Number(CHAIN_ID);
+
+const activeChain =
+  pipe(
+    [mainnet, sepolia],
+    find(chain => chain.id === activeChainId),
+  ) ?? sepolia;
+
+const mainnetRpcUrl = process.env.VITE_MAINNET_RPC_URL || 'https://ethereum-rpc.publicnode.com';
+const sepoliaRpcUrl =
+  process.env.VITE_SEPOLIA_RPC_URL || 'https://ethereum-sepolia-rpc.publicnode.com';
+
+export const config = createConfig({
+  chains: [activeChain],
+  transports: {
+    [mainnet.id]: http(mainnetRpcUrl),
+    [sepolia.id]: http(sepoliaRpcUrl),
+  },
+  connectors: [
+    injected(),
+    walletConnect({
+      projectId: WALLET_CONNECT_V2_PROJECT_ID,
```

Comment:
shouldn't this one be prefixed with `VITE_`?

Reply:
This is a constant. 
https://github.com/nounsDAO/nouns-monorepo/blob/c7c4d6f9039237665182cfc6d5264103bc2f94b6/packages/nouns-webapp/src/config.ts#L54-L55

Reply:
lol my bad

we should probably get rid of those constants, tho. not a fan of needless indirections.
anyway, besides the point of this PR